frontend/src/pages/RegionalGoalDashboard/index.js

```diff
@@ -0,0 +1,173 @@
+import React, { useCallback, useContext, useMemo } from 'react';
+import PropTypes from 'prop-types';
+import { Helmet } from 'react-helmet';
+import { Grid, GridContainer } from '@trussworks/react-uswds';
+import { v4 as uuidv4 } from 'uuid';
+import useSessionFiltersAndReflectInUrl from '../../hooks/useSessionFiltersAndReflectInUrl';
+import { getUserRegions } from '../../permissions';
+import UserContext from '../../UserContext';
+import { expandFilters, formatDateRange } from '../../utils';
+import { buildDefaultRegionFilters, showFilterWithMyRegions } from '../regionHelpers';
+import './index.css';
+import { DASHBOARD_FILTER_CONFIG } from './constants';
+import RegionPermissionModal from '../../components/RegionPermissionModal';
+import FilterPanel from '../../components/filter/FilterPanel';
+import GoalsPercentage from '../../widgets/RegionalGoalDashboard/GoalsPercentage';
+import GoalStatusChart from '../../widgets/RegionalGoalDashboard/GoalStatusChart';
+import TotalHrsAndRecipientGraphWidget from '../../widgets/TotalHrsAndRecipientGraph';
+import TopicsTable from '../../widgets/RegionalGoalDashboard/TopicsTable';
+
+const defaultDate = formatDateRange({
+  lastThirtyDays: true,
+  forDateTime: true,
+});
+
+const FILTER_KEY = 'regional-goal-dashboard-filters';
+
+// TODO: All of the filter logic was directly ripped from RegionalDashboard.
+// This should be refactored.
+export default function RegionalGoalDashboard() {
+  const { user } = useContext(UserContext);
+  const hasCentralOffice = useMemo(() => (
+    user && user.homeRegionId && user.homeRegionId === 14
+  ), [user]);
+  const regions = useMemo(() => getUserRegions(user), [user]);
+  const userHasOnlyOneRegion = useMemo(() => regions.length === 1, [regions]);
+  const defaultRegion = useMemo(() => regions[0].toString(), [regions]);
+
+  const allRegionsFilters = useMemo(() => buildDefaultRegionFilters(regions), [regions]);
+
+  const getFiltersWithAllRegions = () => {
+    const filtersWithAllRegions = [...allRegionsFilters];
+    filtersWithAllRegions.push({
+      id: uuidv4(),
+      topic: 'startDate',
+      condition: 'is within',
+      query: defaultDate,
+    });
+    return filtersWithAllRegions;
+  };
+
+  const centralOfficeWithAllRegionFilters = getFiltersWithAllRegions();
+
+  const defaultFilters = useMemo(() => {
+    if (hasCentralOffice) {
+      return centralOfficeWithAllRegionFilters;
+    }
+
+    return [
+      {
+        id: uuidv4(),
+        topic: 'region',
+        condition: 'is',
+        query: defaultRegion,
+      },
+      {
+        id: uuidv4(),
+        topic: 'startDate',
+        condition: 'is within',
+        query: defaultDate,
+      },
+    ];
+  }, [defaultRegion, hasCentralOffice, centralOfficeWithAllRegionFilters]);
+
+  const [filters, setFiltersInHook] = useSessionFiltersAndReflectInUrl(FILTER_KEY, defaultFilters);
+
+  const setFilters = useCallback((newFilters) => {
```

Comment:
Not sure we need this callback. Was talking to Matt today and it was added for paging.

IE on page 3 then new filters only return 1 page of results.

Reply:
Right, the Regional Dashboard wants some pagination stuff reset back to the defaults when a filter is changed